Tests/Client/Unit Tests/Channel_SilentMessageTests.swift

```diff
@@ -0,0 +1,250 @@
+                        "role": "user",
+                        "created_at": "2019-06-05T15:01:52.847807Z",
+                        "image": "https:////i.imgur.com//EgEPqWZ.jpg",
+                        "updated_at": "2020-05-12T12:13:01.306955Z",
+                        "last_active": "2020-05-12T12:13:01.30335Z",
+                        "name": "steep-moon-9"
+                    },
+                    "last_read": "2020-05-12T12:16:01.14596864Z"
+                }
+            ],
+            "members": [
+                {
+                    "created_at": "2020-05-12T12:16:01.132467Z",
+                    "updated_at": "2020-05-12T12:16:01.132467Z",
+                    "role": "owner",
+                    "user": {
+                        "banned": false,
+                        "online": true,
+                        "id": "broken-waterfall-5",
+                        "role": "user",
+                        "created_at": "2019-03-08T14:45:03.243237Z",
+                        "image": "https:////getstream.io//random_svg//?id=broken-waterfall-5&amp;name=Broken+waterfall",
+                        "updated_at": "2020-05-12T12:22:29.788007Z",
+                        "last_active": "2020-05-12T12:16:00.498889Z",
+                        "name": "broken-waterfall-5"
+                    }
+                },
+                {
+                    "created_at": "2020-05-12T12:16:01.132467Z",
+                    "updated_at": "2020-05-12T12:16:01.132467Z",
+                    "role": "member",
+                    "user": {
+                        "banned": false,
+                        "online": true,
+                        "test": 1,
+                        "id": "steep-moon-9",
+                        "role": "user",
+                        "created_at": "2019-06-05T15:01:52.847807Z",
+                        "image": "https:////i.imgur.com//EgEPqWZ.jpg",
+                        "updated_at": "2020-05-12T12:13:01.306955Z",
+                        "last_active": "2020-05-12T12:13:01.30335Z",
+                        "name": "steep-moon-9"
+                    }
+                }
+            ],
+            "membership": {
+                "created_at": "2020-05-12T12:16:01.132467Z",
+                "updated_at": "2020-05-12T12:16:01.132467Z",
+                "role": "channel_member",
+                "user": {
+                    "banned": false,
+                    "online": true,
+                    "id": "broken-waterfall-5",
+                    "role": "user",
+                    "created_at": "2019-03-08T14:45:03.243237Z",
+                    "image": "https:////getstream.io//random_svg//?id=broken-waterfall-5&amp;name=Broken+waterfall",
+                    "updated_at": "2020-05-12T12:22:29.788007Z",
+                    "last_active": "2020-05-12T12:16:00.498889Z",
+                    "name": "broken-waterfall-5"
+                }
+            }
+        }
+        """
```

Comment:
Creating the object directly wouldn't call `private func calculateUnreadCount`
Since I hardcoded the json response, I don't expect the test to fail _unless_ it needs to (we change the underlying object)
I feel your pain about the long test.. Suggestions welcome!

Reply:
Maybe move it at least to some private variable outside of the test? It's very noisy like this.

Reply:
hmm, i can move it at the end of the test, so it'd look better

Reply:
👍